lib/wda/utils.js

```diff
@@ -8,10 +8,8 @@ import { WDA_RUNNER_BUNDLE_ID } from 'appium-webdriveragent';
 
 
 const PROJECT_FILE = 'project.pbxproj';
-const XCUICOORDINATE_FILE = 'PrivateHeaders/XCTest/XCUICoordinate.h';
 const FBMACROS_FILE = 'WebDriverAgentLib/Utilities/FBMacros.h';
```

Comment:
Do we need these patchers anyway now? Since WDA is under our control it is possible to fix this stuff directly there

Reply:
In the long run it can probably be done there. Right now it still needs to be done here, until that happens.